examples/using-gatsby-image/src/components/floating-image.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import Img from "gatsby-image"
+
+import { rhythm, options } from "../utils/typography"
+
+const FloatingImage = ({ imageMobile, imageDesktop }) => (
+  <React.Fragment>
+    <Img
+      style={{ display: `inherit` }}
```

Comment:
nit: but this may as well go in the css prop (or as the styled component approach, I think both work well)

Reply:
`gatsby-image` sets a couple of inline styles on its outer container and allows overriding via its `style` prop. One of these inline styles is `display: inline-block;`.

In this case we have two images, one for mobile and one for desktop, and toggle their visibility with `display`, so we want to override the `gatsby-image` default to make our own styles work.